app/helpers/youtube_helper.rb

```diff
@@ -0,0 +1,8 @@
+module YoutubeHelper
+  # the embed method will accept a regular youtube url and convert it into an embed url
+  # In our view, we can just call <%= embed(some youtube url) %>
```

Comment:
Weird that this is showing up in your PR? These comments should go away though...